tests/test_compose_resources/docker-compose.yml

```diff
@@ -1,77 +1,77 @@
-    - routes__r2=FROM /messages/modules/SampleModule/outputs/output1 INTO $$upstream
-    - IotHubConnectionString=HostName=HostName;DeviceId=DeviceId;ModuleId=$$edgeHub;SharedAccessKey=SharedAccessKey
-    - EdgeModuleHubServerCAChainCertificateFile=/mnt/edgehub/edge-chain-ca.cert.pem
-    - EdgeModuleHubServerCertificateFile=/mnt/edgehub/edge-hub-server.cert.pfx
-    - configSource=local
-    - SSL_CERTIFICATE_PATH=/mnt/edgehub/
-    - SSL_CERTIFICATE_NAME=edge-hub-server.cert.pfx
-    image: mcr.microsoft.com/azureiotedge-hub:1.1
-    labels:
-      iotedgehubdev: ''
-    networks:
-      azure-iot-edge-dev:
-        aliases:
-        - gatewayhost
-    ports:
-    - 8883:8883/tcp
-    - 443:443/tcp
-    restart: always
-    volumes:
-    - source: edgehubdev
-      target: /mnt/edgehub
-      type: volume
-  tempSensor:
-    container_name: tempSensor
-    depends_on:
-    - edgeHubDev
-    environment:
-    - EdgeModuleCACertificateFile=/mnt/edgemodule/edge-device-ca.cert.pem
-    - EdgeHubConnectionString=HostName=HostName;DeviceId=DeviceId;ModuleId=tempSensor;SharedAccessKey=SharedAccessKey
-    image: mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0
-    labels:
-      iotedgehubdev: ''
-    networks:
-      azure-iot-edge-dev: null
-    restart: on-failure
-    volumes:
-    - source: edgemoduledev
-      target: /mnt/edgemodule
-      type: volume
-  SampleModule:
-    container_name: SampleModule
-    depends_on:
-    - edgeHubDev
-    environment:
-    - FOO=rab
-    - BAZ=quux
-    - test1=
-    - BAR=rua
-    - test2=
-    - EdgeModuleCACertificateFile=/mnt/edgemodule/edge-device-ca.cert.pem
-    - EdgeHubConnectionString=HostName=HostName;DeviceId=DeviceId;ModuleId=SampleModule;SharedAccessKey=SharedAccessKey
-    image: localhost:5000/samplemodule:0.0.1-amd64
-    labels:
-      iotedgehubdev: ''
-    networks:
-      azure-iot-edge-dev: null
-    restart: always
-    volumes:
-    - source: edgemoduledev
-      target: /mnt/edgemodule
-      type: volume
-networks:
-  azure-iot-edge-dev:
-    external: true
-volumes:
-  edgehubdev:
-    name: edgehubdev
-  edgemoduledev:
-    name: edgemoduledev
+version: '3.6'
+services:
+  edgeHubDev:
+    container_name: edgeHubDev
+    environment:
+    - OptimizeForPerformance=false
+    - routes__r1=FROM /messages/modules/tempSensor/outputs/temperatureOutput INTO
+      BrokeredEndpoint("/modules/SampleModule/inputs/input1")
+    - routes__r2=FROM /messages/modules/SampleModule/outputs/output1 INTO $$upstream
+    - IotHubConnectionString=HostName=HostName;DeviceId=DeviceId;ModuleId=$$edgeHub;SharedAccessKey=SharedAccessKey
+    - EdgeModuleHubServerCAChainCertificateFile=/mnt/edgehub/edge-chain-ca.cert.pem
+    - EdgeModuleHubServerCertificateFile=/mnt/edgehub/edge-hub-server.cert.pfx
+    - configSource=local
+    - SSL_CERTIFICATE_PATH=/mnt/edgehub/
+    - SSL_CERTIFICATE_NAME=edge-hub-server.cert.pfx
+    image: mcr.microsoft.com/azureiotedge-hub:1.1
```

Comment:
on edgedev the edgehub version tag here is swappable for both 1.0 and 1.1, do we want to default to 1.1 in unit tests?